setup.py

```diff
@@ -450,7 +450,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     pandas_requirement,
 ]
 psrp = [
-    'pypsrp~=0.5',
+    'pypsrp~=0.8',
```

Comment:
Is 0.8 really necessary here? We usully limit the versions if we know earlier versions will not work - just to avoid too "strict" limits. Is there something that prevents us to use ~0.7 here? Also how about `> 0.8` ?  We have discussion about not limiting upperbound because we do not really know the future and constraints are protecting our users, so limitng upperbound (in this case <1.0) might not be justified at all.